kratos/containers/container_variable_data/expressions/expression.cpp

```diff
@@ -0,0 +1,44 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya
+//
+
+// System includes
+
+// Project includes
+
+// Include base h
+#include "expression.h"
+
+namespace Kratos {
+
+std::size_t Expression::GetFlattenedSize() const
+{
+    IndexType result = 1;
+    for (const auto v : this->GetShape()) {
+        result *= v;
+    }
+    return result;
```

Comment:
```suggestion
    const auto& r_shape = this->GetShape();
    return std::accumulate(r_shape.begin(),
                           r_shape.end(),
                           1UL,
                           [] (auto left, auto right) {return left * right;});
```

you'll need to `#include <numeric>`

Reply:
done :)